pkg/scheduling/requirements_test.go

```diff
@@ -288,6 +288,248 @@ var _ = Describe("Requirements", func() {
 			Expect(lessThan9.Compatible(lessThan1)).To(Succeed())
 			Expect(lessThan9.Compatible(lessThan9)).To(Succeed())
 		})
+		It("should be strictly compatible", func() {
```

Comment:
It's hard to understand what's different here. It would be good to make a comment around what is _actually_ different between this and the above checks on `Compatible()` since (from what I can tell) this set of tests is basically just a copy from above with a few exceptions.